assets/src/dashboard/app/views/savedTemplates/header/index.js

```diff
@@ -46,6 +46,22 @@ function Header({ filter, search, sort, templates, view }) {
     actions: { scrollToTop },
   } = useLayoutContext();
 
+  const searchOptions = useMemo(() => {
```

Comment:
this is for posterity so we can turn on `in progress views` on the dashboard and avoid a crash. Same structure as stories.